src/flat-file-adapter.js

```diff
@@ -0,0 +1,36 @@
+const flatfile = require('flat-file-db')
+const promise = require('promise')
+const dateFns = require('date-fns')
+
+const db = flatfile.sync(process.env.DB_NAME || 'views.db')
+
+module.exports = {
+  put: promise.denodeify(db.put.bind(db)),
+
+  has: (key) => Promise.resolve(db.has(key)),
+  get: (key, options) => {
+    const value = db.get(key) || { views: [] }
+
+    return {
+      views: value.views.filter(view => {
+        if (options && options.before && dateFns.isAfter(view.time, options.before)) return false
+        if (options && options.after && dateFns.isBefore(view.time, options.after)) return false
```

Comment:
I'm wary of introducing a dependency for this. Since it's all UTC (I assume), why can't we just do `view.time < options.before`?

The issue is that by doing this in the default adapter, we require every subsequent adapter to do the same thing to keep the API promise of accepting every input that `dataFns` does. That's additional overhead I don't think we should introduce, but maybe I'm missing something here.